README.md

```diff
@@ -84,26 +84,26 @@ Additionally, we have designed every element of the Stack such that APIs as well
 |                                         Fireworks                                          |         Hosted         | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    |                    |
```

Comment:
@terrytangyuan What formatter are you using?

I'm using the default Vscode formatting and getting different results:

<img width="1091" alt="image" src="https://github.com/user-attachments/assets/44af73ce-3dd4-435f-8d60-2c45155a9574" />

Reply:
Hmm interesting. I used the default markdown plugin in PyCharm.